src/test/java/com/microsoft/sqlserver/jdbc/MaxResultBufferTest.java

```diff
@@ -0,0 +1,280 @@
+/*
```

Comment:
Please add more test for different possible values of the connection property, both valid and invalid cases.